app/Dto/QueryAnimeSeasonCommand.php

```diff
@@ -20,7 +21,13 @@
 
 abstract class QueryAnimeSeasonCommand extends Data implements DataRequest
 {
-    use HasSfwParameter, HasKidsParameter, HasUnapprovedParameter, HasLimitParameter, HasRequestFingerprint, HasPageParameter;
+    use HasSfwParameter,
+        HasKidsParameter,
+        HasUnapprovedParameter,
+        HasLimitParameter,
+        HasRequestFingerprint,
+        HasPageParameter,
+        HasContinuingParameter;
```

Comment:
@pushrbx 
Flags can't be used as flags in the URL. e.g `?sfw`, `?continuing`
In order to use them, they require a value. e.g `?sfw=true`, `?continuing=true`. Otherwise they fallback to `false`.

Reply:
I need to add a test for this then :thinking:

Reply:
@irfan-dahir This now fixed :tada: